README.md

```diff
@@ -51,6 +51,10 @@ You can turn on format-on-save on a per-language basis by scoping the setting:
 
 `eslint`, `tslint`, and all peer dependencies required by your specific configuration must be installed locally.  Global installations will not be recognized.
 
+### Prettier Plugins
+
+Prettier plugins are supported when using a local Prettier installation. Just install the plugin you need locally to your project, restart vscode and you are good to go.
```

Comment:
Does this mean that this will only work on Node projects? My main use-case for this is formatting TOML config files which may not be in a repo that contains a `package.json` and a local instance of prettier - is there a way to get plugins working with the bundled instance of prettier?

Reply:
It currently needs local prettier installation but nothing prevents you from installing it locally to non-node projects.

I don't think it's possible use plugins with the bundled prettier but it could be possible to use global prettier with plugins. I need to investigate this.

Personally don't ever use the global or the bundled prettier. Always a local installation (even in non-js/node projects) to avoid formatting differences between developers because it varies between prettier versions.

Reply:
IMHO global prettier support with plugins is quite important for non-js users. Supporting node_modules is not always better than ensuring that the global prettier + config is of the right version before working on some project.

Reply:
Global support added 🎉 	🎁

Reply:
Nice! I'm glad to hear this, I didn't want to add a package.json+node_modules to all my non-node projects 👍